projects/plugins/crm/includes/ZeroBSCRM.DataIOValidation.php

```diff
@@ -90,45 +90,88 @@ function zeroBSCRM_io_sanitizeInt($input=''){
 
     }
 
-    // takes WP Editor content + applies WP conversion (adds <p> and deals with <b> as if was wp post)
-    // KSES = Strips evil scripts
-    // NOTE: html strings saved in this way should be output via
-    function zeroBSCRM_io_WPEditor_WPEditorToDB( $string = '' ){
+/**
+ * Takes WP Editor content + applies WP conversion (adds <p> and deals with <b> as if was wp post)
+ *
+ * @deprecated $$next-version$$ use wp_kses directly instead.
+ *
+ * @param string $string A string including the HTML content to be saved to the DB.
+ *
+ * @return string
+ */
+function zeroBSCRM_io_WPEditor_WPEditorToDB( $string = '' ) {
 
-        global $zbs;
+	global $zbs;
 
-        return wp_kses( $string, $zbs->acceptable_html );
+	if ( function_exists( '_deprecated_function' ) ) {
```

Comment:
Let's get rid of the `function_exists()`, as it's been around [since WP 2.5](https://developer.wordpress.org/reference/functions/_deprecated_function/#changelog).

Reply:
But also, I think in this case let's chop 'em. They're doing no one any good. :^)

Reply:
I was inspired by the [extra check that WP Super Cache has been implementing ](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/super-cache/wp-cache.php#L139) when using that function. Happy to remove them!

> But also, I think in this case let's chop 'em. 

😱 

All four functions? I'd love to follow the monorepo guidelines, but will also follow your lead on this if that's what you prefer! Maybe we need some more engagement on the post for a team decision?